### PR TITLE
Update figure notebooks to incorporate megan's feedback

### PR DESCRIPTION
**Pull request by taylorreiter**
Updates the notebooks that make the figures in the sourmashconsumr pub to incorporate all of Megan's feedback.

### CODE REVIEW

File: notebooks/20230105_overview_plot.ipynb [hunk 1]
```diff
@@ -4,11 +4,13 @@
    "cell_type": "markdown",
```
Comment (elizabethmcd) [inline notebooks/20230105_overview_plot.ipynb:4]: <p>ah very nice, didn't realize you had made this with code i thought it was hand made</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/2022-sourmashconsumr-validation/pull/4/discussion/#comment-1089284974'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

**Review by elizabethmcd — APPROVED** [review]
Just a few small comments on some of the figures. So beautiful!!

**Comment by taylorreiter** [comment]
<p>meh. this one didn't get included in the pub so I think it might be ok as is</p>

---

 <a href='https://app.reviewnb.com/Arcadia-Science/2022-sourmashconsumr-validation/pull/4/discussion/#comment-1406917634'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by taylorreiter** [comment]
<p>good call, updated!</p>

---

 <a href='https://app.reviewnb.com/Arcadia-Science/2022-sourmashconsumr-validation/pull/4/discussion/#comment-1406917585'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Pull request merged**